### PR TITLE
Remove Poetry

### PR DESCRIPTION
**Pull request by tristan957**
The Python ecosystem is horrible and depending on it for anything is
entirely futile. hse-python will ditch all of its dependencies on the
Python ecosystem as should HSE.

Signed-off-by: Tristan Partin <tpartin@micron.com>

## Issue(s) Addressed
NFSE-5058

### CODE REVIEW

**Comment by tristan957** [comment]
This PR is pending https://github.com/hse-project/hse-python/pull/10

**Review by beckerg — APPROVED** [review]

**Comment by bhaveshvasandani** [comment]
I personally wouldn't like to get the poetry environment removed. But if this has to go in, we need to make sure our internal infrastructure is also rid of using this.

File: README.md [hunk 1]
```diff
@@ -66,28 +61,16 @@ cd hse
 git checkout <release tag>
 ```
 
-Install Python dependencies using Poetry.
-
-```shell
-poetry install
-```
-
 Build and install using Meson and Ninja from within the Poetry shell.
```
Comment (smoyergh) [inline README.md:64]: Remove "from within the Poetry shell".

**Comment by tristan957** [comment]
Bhavesh with these changes, the only real issue is getting Cython >= 0.29. What reasons do you see for keeping it.

**Review by bhaveshvasandani — APPROVED** [review]

**Pull request merged**